content/command-line/concepts/bash/bash.md

```diff
@@ -155,3 +155,94 @@ mv oldFileName.txt newFileName.txt
 # Move a file to a different directory (source path is the first argument, destination path is the second)
 mv sourceDirectory/filename.txt destinationDirectory/filename.txt
 ```
+
+## Bash Functions
+
+A Bash function is essentially a set of commands that can be called numerous times. The purpose of a _function_ is to help you make 
+your bash scripts more readable, and avoid writing the same code repeatedly.
```

Comment:
Avoid using 'you'/'your'. Also, it can be simplified as - 

```suggestion
A **Bash function** is a reusable set of commands that enhances script readability and prevents code duplication.
```